presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemoryMetadata.java

```diff
@@ -300,6 +300,42 @@ public void testCreateTableAndViewInNotExistSchema()
         assertEquals(metadata.listSchemaNames(SESSION), ImmutableList.of("default"));
     }
 
+    @Test
+    public void testRenameTable()
+    {
+        SchemaTableName tableName = new SchemaTableName("test", "test_rename");
```

Comment:
nit: you could use some more meaningful test data. Instead of `test` you use `test_schema`, `test_rename` to `test_table_to_be_renamed`. That way it it is is a bit easier to understand the test logic.